docs/update_and_migration/from_4.0/to_4.1.md

```diff
@@ -166,6 +165,20 @@ Apply the following database update scripts:
     psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-4.1.0-to-4.1.1.sql
     ```
 
+### v4.1.6
+
+#### VCL configuration for Fastly
+
+Ibexa DXP now supports Fastly shielding. If you are using Fastly and want to use shielding, you need to update your VCL files.
```

Comment:
I think we should tell the users to upgrade nevertheless if they are planning to use shielding or not

```suggestion
The .vcl configuration files for Fastly has changed. Follow the upgrade description below to update then
```

Reply:
```suggestion
The Fastly `.vcl` configuration files have changed. Follow the upgrade steps below to update them.
```